proposals/lip-0071.md

```diff
@@ -281,18 +303,18 @@ This function is used to estimate the expected rewards for a validator assuming
 ##### Execution
 
 ```python
-def getExpectedValidatorRewards(validatorAddress: Address) -> tuple[uint64, uint64, uint64, uint64]:
+def getExpectedValidatorRewards(validatorAddress: Address, height: uint32) -> tuple[uint64, uint64, uint64, uint64]:
```

Comment:
This function is defined so that it can be used in the wallet, to give an estimation on rewards. Therefore, we need an interface for Lisk Desktop to request the expected rewards.  Since `getDefaultRewardAtHeight` depends on the height, it should be correctly called by the wallet depending on the current height.